Added option of multiple messy bib files

ilsePit opened the pull request:
Line 6: changed description messy_bibliography parameter
Line 23: changed to allow for messy_bibliography to be a list of characters
Line 54-57: changed to allow for multiple messy bib files to be used as input in the function tidy_bib_file(), similar to how multiple Rmd files are possible as input.

ilsePit committed:
Added option of multiple messy bib files

Line 6: changed description messy_bibliography parameter
Line 23: changed to allow for messy_bibliography to be a list of characters
Line 54-57: changed to allow for multiple messy bib files to be used as input in the function tidy_bib_file(), similar to how multiple Rmd files are possible as input.

crsh commented:
Again, thanks! This should be very useful. Same question as in the other PR: Does this work if multiple Rmd files are as `c("file1.Rmd", "file2.Rmd")` and `list("file1.Rmd", "file2.Rmd")`? If the latter does not work, it may make sense to slightly change the wording in the documentation to "One path or a vector of paths"?

ilsePit committed:
Update tidy_bib_file.R

changed wording in line 6 from 'list' to 'vector' to properly reflect the fact that the input for messy_bibliography should be in the form of either "messy_references.bib" or c("messy1.bib", "messy2.bib")

ilsePit commented:
you're right, changed it! thanks!

crsh commented:
Thanks once more. Greatly appreciated!

crsh committed:
Merge branch 'master' into patch-4
